src/metadata_tool/tools/reports.clj

```diff
@@ -66,19 +66,67 @@
                             :body     [{ :type    "text/html; charset=\"UTF-8\""
                                          :content body }] } )))
 
+(defn- activity-stale?
+  [activity stale-date]
+  (let [program-id (:program cfg/config)
+        activity-date (tf/parse (tf/formatters :date) (:contribution-date activity))
+        compare-date (compare stale-date activity-date)]
+    (and (pos? compare-date) (= "INCUBATING" (:state activity)))))
+
 (defn- send-email-to-pmc
   [program-id subject body]
-  (if-not (s/blank? program-id)
-    (send-email (:pmc-mailing-list-address (md/program-metadata program-id))
-                subject
-                body
-                :cc-addresses     program-liaison-email-address
-                :reply-to-address program-liaison-email-address)))
+  ; TODO - enable code below and comment out print commands!
+  (println "===========================")
+  (println subject)
+  (println "---------------------------")
+  (println body)
+  (println "==========================="))
+  ; (if-not (s/blank? program-id)
+    ; (send-email (:pmc-mailing-list-address (md/program-metadata program-id))
+    ;             subject
+    ;             body
+    ;             :cc-addresses     program-liaison-email-address
+    ;             :reply-to-address program-liaison-email-address)))
+
+(defn- assoc-org-name
+  [person]
+  (let [id (:person-id person)]
+    (assoc person :org-name (or (str " (" (:organization-name (first (md/current-affiliations id))) ")") ""))))
+
+(defn- activities
```

Comment:
This probably belongs in the ```metadata-tool.sources.metadata``` namespace, since:

1. It's useful for more than just the reports.
2. It hardcodes metadata constants (i.e. ```ARCHIVED```) that may change in future - it's easier to deal with such changes if they're isolated to one ns (encapsulation).